### PR TITLE
Bugfix Shells

### PR DESCRIPTION
**Pull request by philbucher**
Fixes the problems in #915 

The test I uncommented is failing with the new processes, it was working with the deprecated ones.

I am trying to debug it, will most likely include it in another PR

the stuff uncommented in the cpp's is only needed for the composite laws which are anyway only added in #842

### CODE REVIEW

**Comment by RiccardoRossi** [comment]
@loumalouomega can u take a look if it fixes #915 ?

**Review by RiccardoRossi — APPROVED** [review]
looks ok to me. Still i would like the green light of @loumalouomega

**Comment by RiccardoRossi** [comment]
uff, i did some corrections and pushed a change. With that change all of the tests work in this branch.

**Comment by philbucher** [comment]
Hi Riccardo 
It should fix #915, the tests on my machine passed in FullDebug

File: applications/StructuralMechanicsApplication/tests/test_loading_conditions.py [hunk 1]
```diff
@@ -218,9 +281,16 @@ def _test_SurfaceLoadCondition3D4N(self):
             self.assertAlmostEqual(rhs[i],reference_res[i])     
         
     def test_execution(self):
+<<<<<<< HEAD
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/tests/test_loading_conditions.py]: Be careful, this is a conflict, remove it

**Review by loumalouomega — CHANGES_REQUESTED** [review]

**Review by loumalouomega — APPROVED** [review]

**Review by loumalouomega — CHANGES_REQUESTED** [review]
It doesn't compile

**Comment by loumalouomega** [comment]
~~~
/home/vicente/bin/Kratos/applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp: In member function ‘virtual void Kratos::LineLoadCondition2D::CalculateAll(Kratos::Condition::MatrixType&, Kratos::Condition::VectorType&, Kratos::ProcessInfo&, bool, bool)’:
/home/vicente/bin/Kratos/applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp:228:38: error: ‘class Kratos::LineLoadCondition2D’ has no member named ‘CalculateAndAddWorkEquivalentNodalForcesLineLoad’
         if (this->HasRotDof()) this->CalculateAndAddWorkEquivalentNodalForcesLineLoad(gauss_load,rRightHandSideVector);
                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/vicente/bin/Kratos/applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp:228:87: error: ‘gauss_load’ was not declared in this scope
         if (this->HasRotDof()) this->CalculateAndAddWorkEquivalentNodalForcesLineLoad(gauss_load,rRightHandSideVector);
                                                                                       ^~~~~~~~~~
/home/vicente/bin/Kratos/applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp:228:87: note: suggested alternative: ‘getpayload’
         if (this->HasRotDof()) this->CalculateAndAddWorkEquivalentNodalForcesLineLoad(gauss_load,rRightHandSideVector);

~~~

**Comment by loumalouomega** [comment]
@philbucher and @RiccardoRossi Travis can not compile too

**Comment by RiccardoRossi** [comment]
ok, it passed now.

can u merge it? (if you agree)

**Review by loumalouomega — APPROVED** [review]

**Comment by loumalouomega** [comment]
Ok

**Pull request merged**